data_generators/xcspec_extractor.py

```diff
@@ -35,20 +35,14 @@ def xcspec_from_file(path):
 xcspec_bzl_path = sys.argv[3]
 eval_bzl_path = sys.argv[4]
 xcode = os.path.dirname(os.path.dirname(developer_dir))
-xcode3plusgins = os.path.join(
-    xcode, 'Contents/PlugIns/Xcode3Core.ideplugin/Contents/SharedSupport/Developer/Library/Xcode/Plug-ins')
+xcb_specifications = os.path.join(xcode, 'Contents/PlugIns/XCBSpecifications.ideplugin/Contents/Resources')
```

Comment:
@luispadron thanks for adding this 👏 

The main problem with the current implementation of `xcspecs.bzl` is that it isn't switched on `Xcode` versions. In other words, this data should be generated dependent on the Xcode version ( ideally by Bazel instead of ad-hoc ).

Someone will need to address the non trivial refactoring to pull it of. This PR is fine for now, we should verify that this isn't causing a regression for older Xcodes. Worst case if you're not on 13.x.x you'll need to fork it or fix these issues.

Reply:
Was making this change a depenency of `13.4.1` - that the file is generated by a given version will implicate different flags. 

It's a pretty big change - the other thing, is if you wanted to have simpler solution for now, we can move `xcspecs.bzl` to each version level `xcspecs_13.3.1.bzl` and have the ability to override it in the macros as a stopgap